zstacklib/zstacklib/utils/lvm.py

```
@@ -2743,3 +2763,93 @@ def check_lvmlockd_log():
+                  {"pv_name":"/dev/sdb", "vg_name":"d7ac3dafa12f4ac2aeace05f279cd010"},
+                  {"pv_name":"/dev/sdc", "vg_name":"7c32b42292e648cf92ef61b8b48bb41f"},
+                  {"pv_name":"/dev/sdd", "vg_name":"3c93394b20b14db892fa3be01e3cbf54"},
+                  {"pv_name":"/dev/vda2", "vg_name":"zstack"}
+              ]
+          }
+      ]
+    }
+
+     {'/dev/sdd': ['3c93394b20b14db892fa3be01e3cbf54'], '/dev/sdb': ['d7ac3dafa12f4ac2aeace05f279cd010'], '/dev/sdc': ['7c32b42292e648cf92ef61b8b48bb41f'], '/dev/vda2': ['zstack']}
+    '''
+    r, o, e = bash.bash_roe("pvs --reportformat json -o pv_name,vg_name")
```

Ruff (0.14.14)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)